simulator/src/circuit.js

```
@@ -153,7 +153,29 @@ export function createNewCircuitScope() {
-    name = name || prompt('Enter circuit name:', 'Untitled-Circuit');
+
+    if (name) {
+        return finalizeNewCircuit(name, id, isVerilog, isVerilogMain);
+    }
+
+    $('#circuitNameInput').val('Untitled-Circuit');
+    $('#circuitNameDialog').dialog({
+        resizable: false,
+        modal: true,
+        buttons: {
+            'OK': function() {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)